definitions/ext-rf_scanner/definition.yml

```diff
@@ -21,20 +21,23 @@ synthesis:
   conditions:
     # The attribute’s value must match the provided value
     - attribute: eventType
+
       prefix: "THD_RF_TRANSACTION"
 
+
   # Telemetry attributes that should be extracted into entity tags.
   tags:
     ipAddr:
     osDetails:
+    macAddress:
+    serialNumber:
     distributionCenter:
-    inCradle:
-      multiValue: false
+    lastActiveTimestamp:
```

Comment:
this sounds like `multiValue: true` right?
every time it changes we will have a new tag. This will explode the number of tags we store.

Also i would bee keen on not adding if it's not really required. Since even with only 1 value it will mean a bunch of updates on the system.

Reply:
the lastActiveTimestamps is not  multivalue, in fact all the tags should be  multivalue: false

Reply:
The default is `multiValue: true`, so then we should set them to false.

Reply:
sorry i was sugesting to make the multiValue:false but my brain betrayed me :D